pages/project/[id].page.js

```diff
@@ -53,7 +53,7 @@ const ProjectPage = () => {
     watch(PLEDGE_AMOUNT)
   );
 
-  const isOwner = address === campaign?.owner?.publicAddress;
+  const isOwner = address === campaign?.campaign?.owner?.publicAddress;
```

Comment:
@lucasgalarce , did we change the response for that endpoint that was returning `{ data: { campaign: { campaign }, pledges }}`?

Reply:
Yes, is as Rena said. I guess that the latest version of api isn't deployed. I'm deploying again now